client/components/place/placesOnMap.factory.js

```diff
@@ -94,16 +109,16 @@ angular.module('mapModule')
         coords[1] = place.location.coordinates[0];
         coordsArray[index] = coords;
       });
-      trackForAdding = polyline(coordsArray, color).addTo(map);
+      trackForAdding = polyline(coordsArray, color).addTo(placesOnMap.map);
       tracks.push([trackForAdding, track.type]);
     };
 
     var removeTrack = function(track) {
       if (this == 'all') {
```

Comment:
It isn`t strange because I use removeTrack function in forEach method (line 135) where second parameter refers to `this` of first parameter which is function (in this case it is removeTrack function), so I can assign to `this` whatever I want: true, 'all', 'allTracks'. etc. In this case I think using 'all' is more clear then true.